pkgs/by-name/sy/syncyomi/package.nix

```diff
@@ -0,0 +1,105 @@
+{ lib
+, stdenvNoCC
+, fetchFromGitHub
+, buildGoModule
+, jq
+, moreutils
+, nodePackages
+, cacert
+, esbuild
+}:
+
+buildGoModule rec {
+  pname = "syncyomi";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "SyncYomi";
+    repo = "SyncYomi";
+    rev = "refs/tags/v${version}";
```

Comment:
You do not need to specify `refs/tags/v` for it to pull a tag.

Reply:
In my original PR I was advised to add it to be explicit.

Reply:
@RossComputerGuy `fetchFromGitHub` does not figure anything, github itself does. and this avoids ambiguity when there are identically named tags and branches, for example.